SampleApp-Objc/AppDelegate.m

```diff
@@ -24,7 +24,7 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
 }
 
 - (NSString *)getEnvironmentVariableForKey:(NSString *)key {
-  if (NSBundle.mainBundle.infoDictionary[key]) {
+  if (NSBundle.mainBundle.infoDictionary[key] && (NSString*)NSBundle.mainBundle.infoDictionary[key] != [NSString stringWithFormat:@""]) {
```

Comment:
Oh bother. That's my Swift showing - swift overloads != and == for strings to do proper comparison.

Yeah either length check or using `isEqualToString` although i think a length check here is more readable.